HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -144,6 +144,12 @@ def customiseFor2018Input(process):
     return process
 
 
+#temporary solution to add GEM geometry for hltGetConfiguration
+def customiseForRun3GEMGeometry34788(process):
+    """Add GEM geometry to output from hltGetConfiguration"""
+    process.load("Geometry.GEMGeometryBuilder.gemGeometryDB_cfi")
+
```

Comment:
If this function needs to be used via `hltGetConfiguration --customise [..].customiseForRun3GEMGeometry34788` (which will write `process = customiseForRun3GEMGeometry34788(process)`), then the function body is missing `return process`.

In general, if the function is run from inside `customizeHLTforCMSSW`, passing it via the `--customise` option of `hltGetConfiguration` is not needed; if that's done, the function will be run twice (first by `customizeHLTforCMSSW`, then by the call added via `--customise`).

Also (but this is just an opinion), if the function is meant to be temporary, its name should just contain the PR number (i.e. `customiseFor34788`), plus a description as comment in the function body (which is already done). If the function is not temporary (but this is a much less common occurrence), then one should probably not use the PR number, like for other functions in this file (e.g. `customisePixelGainForRun2Input`).

Reply:
Fixed. Thx.